README.md

```diff
@@ -2,27 +2,6 @@
 [![CircleCI](https://circleci.com/gh/argoproj/argo.svg?style=svg)](https://circleci.com/gh/argoproj/argo)
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=argoproj_argo&metric=alert_status)](https://sonarcloud.io/dashboard?id=argoproj_argo)
 
-# Argoproj - Get stuff done with Kubernetes
```

Comment:
I think we should leave the logo for now until people get used to the new structure and we replace it with something specific to workflows.